forms/AddressLookup/index.js

```diff
@@ -126,6 +126,12 @@ export default React.createClass({
     })
   },
 
+  handleCountrySelectBlur() {
+    this.setState({
```

Comment:
Is this worth unit testing?  Doesn't look like we have many tests for this file anyways